### DISCUSSION TITLE
woocommerce recently viewed products

### DISCUSSION THREAD

**Discussion by ludwigPop** [bootscore/bootscore#875] [Q&A]
Hi, 
I need your your help. I have a display problem with the woocommerce_recently_viewed_products widget. It appears perfectly in the pages of the store. But on the other pages, the display is broken. I'm short on the problem. Does anyone have an idea to fix this bug?

**Comment by crftwrk** [comment]
Hi,

please share a link to a page where the widget is fine and a link to a page where the widget is broken. We have to check and debug this in the browser.

**Comment by ludwigPop** [comment]
Hi ! Thanks for your response. I solved my problem by adding this code to my stylesheet:

```
ul.product_list_widget {
    padding: 0;
}

ul.product_list_widget li {
    padding: 4px 0;
    margin: 0;
    list-style: none;
}

ul.product_list_widget li a {
    display: block;
    font-weight: 700;
}

ul.product_list_widget li img {
    float: right;
    margin-left: 4px;
    width: 32px;
    height: auto;
    box-shadow: none;
}


```
